oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/TimeSeriesUtils.java

```diff
@@ -60,6 +61,31 @@ public static String latestWriteIndexName(Model model) {
         }
     }
 
+    /**
+     * @return Concrete index name for super dataset index
+     */
+    public static String[] querySuperDatasetIndices(String indexName, long startTimeBucket, long endTimeBucket) {
```

Comment:
I think this should be renamed to `superDatasetIndexName`, right?

Reply:
ok